src/gt4sd/algorithms/generation/tests/test_hugging_face.py

```diff
@@ -170,6 +179,11 @@ def test_available_versions(config_class: Type[AlgorithmConfiguration]):
             HuggingFaceGenerationAlgorithm,
             marks=pytest.mark.skipif(test_settings.gt4sd_ci, reason="high_memory"),
         ),
+        pytest.param(
+            HuggingFaceT5Generator,
+            HuggingFaceGenerationAlgorithm,
+            marks=pytest.mark.skipif(test_settings.gt4sd_ci, reason="high_memory"),
```

Comment:
I would not skip the test

```suggestion
            marks=pytest.mark.skipif(test_settings.gt4sd_ci, reason="high_memory")
```